src/core/midiPorts.h

```diff
@@ -81,6 +81,9 @@ std::vector<std::string> getInPorts(bool addr = false);
 // Sends a MIDI message 'mm' to a port named 'recipient'.
 void midiReceive(const MidiMsg& mm, const std::string& recipient);
 
+// shortName
+// Returns shortened port name (for representation in tight spaces)
+std::string shortName(const std::string& port);
```

Comment:
I see, this is the comment I was expecting above.
`// shortName` is probably useless.

Reply:
It is, I just followed the convention from old kernelMidi.h, I guess.

Reply:
That makes sense then.